packages/cfd/src/Containers/cfd-password-manager-modal.tsx

```diff
@@ -283,15 +299,15 @@ const CFDPasswordManagerModal = ({
     selected_account_group,
     selected_server,
     sendVerifyEmail,
-}) => {
+}: TCFDPasswordManagerModal) => {
     const multi_step_ref = React.useRef();
```

Comment:
don't we need type for `multi_step_ref` ref and `index` ?

Reply:
@hirad-rewok I suppose we need any type for the variable of React.useRef() anyway regarding this article https://linguinecode.com/post/how-to-use-react-useref-with-typescript

Reply:
I've added types for these 2.